### PR TITLE
Script for teamcity tests checking

### PR DESCRIPTION
**Pull request by olga24912**

### CODE REVIEW

File: sd/test/teamcity.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import sys
+import os
+import yaml
+import shutil
+import argparse
+import subprocess
+from traceback import print_exc
+
+#Log class, use it, not print
+class Log:
```
Comment (seryrzu) [inline sd/test/teamcity.py:12]: is this the log class that is expected to be used everywhere in the project? We already have a logger here: https://github.com/ablab/stringdecomposer/blob/devel/sd/standard_logger.py
Can you please merge them?

File: sd/test/teamcity.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    text = ""
+
+    def log(self, s):
+        self.text += s + "\n"
+        print(s)
+        sys.stdout.flush()
+
+    def warn(self, s):
```
Comment (seryrzu) [inline sd/test/teamcity.py:20]: I believe python already provides logger levels. Please see logging.INFO etc

File: sd/test/teamcity.py (hunk 1 above)
Comment (seryrzu) [reply sd/test/teamcity.py:12]: Example of usage: https://github.com/ablab/stringdecomposer/blob/api/sd/cluster_sequences.py

**Review by seryrzu — CHANGES_REQUESTED** [review]

**Review by TanyaDvorkina — COMMENTED** [review]
Looks Ok for me. Let's merge after addressing Andrey's comments.

**Comment by seryrzu** [comment]
This PR became obsolete. CI is currently implemented via GitHub actions.

**Comment by asl** [comment]
Please note that we're only having 2000 runtime minutes for action per month for the whole `ablab` organization.

**Pull request closed without merging**